Configuration/StandardSequences/python/AlCaHarvesting_cff.py

```diff
@@ -167,6 +168,15 @@
                                           timetype   = cms.untracked.string('runnumber')
                                           )
 
+# --------------------------------------------------------------------------------------
+# SiPixel Alignment HG
+ALCAHARVESTSiPixelAliHG_metadata = cms.PSet(record = cms.untracked.string('TrackerAlignmentRcd'))
+
+ALCAHARVESTSiPixelAliHG_dbOutput = cms.PSet(record = cms.string('TrackerAlignmentRcd'),
+                                          tag = cms.string('SiPixelAli_pclHG'),
+                                          timetype   = cms.untracked.string('runnumber')
+                                          )
```

Comment:
I think using `TrackerAlignmentRcd` here might conflict with the key used in the metadata for the current SiPixel Alignment. I think it should be renamed.

Also, related to this, we should update the `DropboxMetadata` code (and the tags) in https://github.com/cms-sw/cmssw/blob/master/CondFormats/Common/test/ProduceDropBoxMetadata.py. @dmeuser do you want to include the changes in this PR or do you want us to do it in a different PR?

Reply:
> do you want to include the changes in this PR or do you want us to do it in a different PR?

As you prefer, if you want me to take care of it I would have the following question:
Do we need a new record? Or is renaming the one used in `AlCaHarvesting_cff.py` enough?
And for the changes in `DropboxMetadata` and in the unit test, can I just duplicate the LG ones and put the corresponding HG parts there?

Reply:
Sorry for the delay in the answer, other committments 😉 

As Marco said, no you don't need a new record, that's not really a record, but rather the key used in the DropboxMetada dictionary, that's why you need to be consistent also when changing the DropboxMetada code.
So you can make a change as simple as:
`TrackerAlignmentRcd` -> `TrackerAlignmentRcd_HG`

About the DropboxMetadata you need to update:
 - [ProduceDropBoxMetadata.py](https://github.com/cms-sw/cmssw/blob/master/CondFormats/Common/test/ProduceDropBoxMetadata.py)
  - [DropBoxMetadataReader.py](https://github.com/cms-sw/cmssw/blob/master/CondFormats/Common/test/DropBoxMetadataReader.py)
  - add the needed json files in [https://github.com/cms-sw/cmssw/tree/master/CondFormats/Common/data](CondFormats/Common/data)
     - in these json files you specify the name (and destination DB) of the tag to which PCL will append the new payloads
     - please note each workflow has 2 json files: one for prod and one for prep DB

Once that is done we (AlCa) will take care of creating the new DropboxMetada payload and append it to the data-taking tags before putting this PCL wf in production.

Reply:
I hope I managed to find all the necessary changes, which are applied in https://github.com/cms-sw/cmssw/pull/38449/commits/b946b6b032e8f879aec3562daf2c055c00d15d5b